doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -217,6 +217,7 @@ The exhaustive list of extensions is available at https://docs.openstack.org/api
 | cold_migration           | X             |
 | console_output           | X             |
 | disk_config              | X             |
+| instance_password        | X             |
```

Comment:
This file changeset is different that what PR indicates. Is it an intended change to be cherry-picked?

Reply:
Sorry I;ve selected the wrong base branch. Now it should be okay.